### ISSUE TITLE
Fix permissions on github token in openvex action

### ISSUE THREAD

**Issue by psschwei** [kubernetes-sigs/kueue#1478]
`#SecuritySlam`

**What would you like to be cleaned**:

From [CLOMonitor](https://clomonitor.io/projects/cncf/kueue#kueue_security):
```
Token-Permissions OpenSSF Scorecard check

Score: 0 (check passes with score >= 5)

Reason: detected GitHub workflow tokens with excessive permissions

Details:

Warn: no topLevel permission defined: .github/workflows/openvex.yaml:1: Visit https://app.stepsecurity.io/secureworkflow/kubernetes-sigs/kueue/openvex.yaml/main?enable=permissions
Tick the 'Restrict permissions for GITHUB_TOKEN'
Untick other options
NOTE: If you want to resolve multiple issues at once, you can visit https://app.stepsecurity.io/securerepo instead. (Low effort)
```

**Why is this needed**:

This shows as a failing test on CLOMonitor, so fixing it will both improve the security of the project and improve the CLOMonitor score.

/sig release

**Comment by ArangoGutierrez** [comment]
This is a result report of the #securitySlam
cc @alculquicondor @tenzen-y

**Comment by psschwei** [comment]
Yes, I can fix that. Since I've already got #1475 open, I'll do it there. I'll also make a similar change in #1467

**Comment by psschwei** [comment]
Commit https://github.com/kubernetes-sigs/kueue/pull/1475/commits/893ac951b23bd65eaaf495f00479ca2a2cf1ca3d should resolve the token permissions issue

**Comment by alculquicondor** [comment]
This was actually fixed in #1475
